gen3/bin/kube-setup-aws-es-proxy.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#
+# Deploy aws-es-proxy into existing commons
+# https://github.com/abutaha/aws-es-proxy
+# 
+
+if [ $# -ne 1 ]; then
+    echo "USAGE: $0 name_of_child_vpc"
+    exit 1
+fi
+
+source "${GEN3_HOME}/gen3/lib/utils.sh"
+gen3_load "gen3/lib/kube-setup-init"
+
+gen3 kube-setup-secrets
+#gen3 roll esproxy
+sed "s/ES_ENDPOINT/https:\/\/${1}/" "${GEN3_HOME}/kube/services/aws-es-proxy/aws-es-proxy-deploy.yaml" | g3kubectl apply -f -
+g3kubectl apply -f "${GEN3_HOME}/kube/services/netpolicy/networkpolicy_aws_es_proxy.yaml"
```

Comment:
So this expects the ES-PROXY endpoint to be passed as a commond line argument to `kube-setup-aws-es-proxy`?

Reply:
Yes, I guess I could try querying with awscli, going to try something out.

This is what's outputed by tfapply, you would need the es_endopoint.

```
Apply complete! Resources: 4 added, 0 changed, 0 destroyed.

Outputs:

es_endpoint = vpc-fauziv1-gen3-metadata-kydadmgkzectkhr2pdd2s2uihu.us-east-1.es.amazonaws.com
kibana_endpoint = vpc-fauziv1-gen3-metadata-kydadmgkzectkhr2pdd2s2uihu.us-east-1.es.amazonaws.com/_plugin/kibana/
Backing up files to s3
Backing up config.tfvars to s3://cdis-state-ac707767160287-gen3/fauziv1_es/config.tfvars
upload: ./config.tfvars to s3://cdis-state-ac707767160287-gen3/fauziv1_es/config.tfvars
Backing up backend.tfvars to s3://cdis-state-ac707767160287-gen3/fauziv1_es/backend.tfvars
upload: ./backend.tfvars to s3://cdis-state-ac707767160287-gen3/fauziv1_es/backend.tfvars
Backing up README.md to s3://cdis-state-ac707767160287-gen3/fauziv1_es/README.md
upload: ./README.md to s3://cdis-state-ac707767160287-gen3/fauziv1_es/README.md
```